yaml.properties

```diff
@@ -7,6 +7,12 @@ publicVersions=1.4.0
 defaults.mavenGroupId=com.github.sbaudoi
 defaults.mavenArtifactId=sonar-yaml-plugin
 
+1.4.1.description=Fixes 3 important issues that could cause analyses to break.
+1.4.1.sqVersions=[6.7,LATEST]
+1.4.1.date=2019-03-10
+1.4.1.changelogUrl=https://github.com/sbaudoin/sonar-yaml/releases/tag/v1.4.1
+1.4.1.downloadUrl=https://github.com/sbaudoin/sonar-yaml/releases/download/v1.4.1/sonar-yaml-plugin-1.4.1.jar
+
 1.4.0.description=Adds a rule that allows developers to check the consistency of quoted strings.
 1.4.0.sqVersions=[6.7,LATEST]
```

Comment:
Only one version can be compatible with `LATEST` at a time. You need to edit this range to end at 7.6 (we sub-in the current version by convention).